logs_ingest/mapping.py

```diff
@@ -31,6 +31,13 @@
     4: 'Informational'
 }
 
+severity_to_log_level_dict = {
```

Comment:
to keep this dict consistent with 'log_level_to_severity_dict' you can replace it with:
`severity_to_log_level_dict = {v: k for k, v in log_level_to_severity_dict.items()}`